src/messages/en.json

```diff
@@ -1,7 +1,47 @@
 {
     "Index": {
-      "title": "Home",
-      "description": "Testing internationalization."
+        "title": "Home",
+        "mission": {
+            "title": "Our Mission",
+            "p1": "Paul Hom Asian Clinic is a 501C(3) non-profit organization committed to serving underserved and uninsured Asian populations in a linguistically and culturally sensitive manner.",
+            "p2": "At Paul Hom we provide free primary care to the medically underserved, quality community medicine for our local community and interpretation services for the underserved Asian community.",
+            "p3": "Our success is a result of the collaborative effort of all our health professional and undergraduate staff working to provide such important services."
```

Comment:
Not a big deal, but I would just suggest changing the `p#` names to something more specific like `paragraph#`

Reply:
Ohhh, yeah that's true p can be a bit vague